Stripe/STPAddCardViewController.m

```diff
@@ -524,8 +545,27 @@ - (void)switchTableViewCell:(STPSwitchTableViewCell *)cell didToggleSwitch:(BOOL
 
 #pragma mark - UITableView
 
+#if DEBUG
+- (void)setForceEnableRememberMeForTesting:(BOOL)forceEnableRememberMeForTesting {
```

Comment:
It's used by tests to enable remember me and set up some ui state so the snapshots can see the remember me strings. If you have a more graceful suggestion for setting up the tests I can change it.